### ISSUE TITLE
Mail.ru sync stopped working

### ISSUE THREAD

**Issue by Dabuka** [rclone/rclone#4616]
#### What is the problem you are having with rclone?
Mail.ru sync stopped working a couple of weeks ago. Seems like mail.ru responds with 403 for file creation operation. Other operations work fine.

#### What is your rclone version (output from `rclone version`)
```
rclone --version
rclone v1.53.1
- os/arch: linux/amd64
- go version: go1.15
```

#### Which OS you are using and how many bits (eg Windows 7, 64 bit)
Ubuntu 18.04.5 LTS 64-bit

####  Which cloud storage system are you using? (eg Google Drive)
Mail.ru

#### The command you were trying to run (eg `rclone copy /tmp remote:tmp`)
rclone touch mailrue:backup/1.txt

#### A log from the command with the `-vv` flag (eg output from `rclone -vv copy /tmp remote:tmp`)
```
rclone touch mailrue:backup/1.txt -vv
2020/09/20 00:52:07 DEBUG : rclone: Version "v1.53.1" starting with parameters ["rclone" "touch" "mailrue:backup/1.txt" "-vv"]
2020/09/20 00:52:07 DEBUG : Creating backend with remote "mailrue:backup/"
2020/09/20 00:52:07 DEBUG : Using config file from "/home/dabuka/.config/rclone/rclone.conf"
2020/09/20 00:52:08 DEBUG : Creating backend with remote "mailru:backup/hrvj2jq72i0s6s6o5am1khefn8"
2020/09/20 00:52:08 DEBUG : [backup/hrvj2jq72i0s6s6o5am1khefn8]: new meta server: https://cld-extapi3.datacloudmail.ru/meta/
2020/09/20 00:52:08 DEBUG : [backup/hrvj2jq72i0s6s6o5am1khefn8]: new upload shard: https://cld-upload8.cloud.mail.ru/upload-web/
2020/09/20 00:52:08 ERROR : Attempt 1/3 failed with 1 errors and: server error 403 (<html>
<head><title>403 Forbidden</title></head>
<body bgcolor="white">
<center><h1>403 Forbidden</h1></center>
<hr><center>nginx/1.14.2</center>
</body>
</html>
)
2020/09/20 00:52:08 ERROR : Attempt 2/3 failed with 1 errors and: server error 403 (<html>
<head><title>403 Forbidden</title></head>
<body bgcolor="white">
<center><h1>403 Forbidden</h1></center>
<hr><center>nginx/1.14.2</center>
</body>
</html>
)
2020/09/20 00:52:08 ERROR : Attempt 3/3 failed with 1 errors and: server error 403 (<html>
<head><title>403 Forbidden</title></head>
<body bgcolor="white">
<center><h1>403 Forbidden</h1></center>
<hr><center>nginx/1.14.2</center>
</body>
</html>
)
2020/09/20 00:52:08 DEBUG : 11 go routines active
2020/09/20 00:52:08 Failed to touch: server error 403 (<html>
<head><title>403 Forbidden</title></head>
<body bgcolor="white">
<center><h1>403 Forbidden</h1></center>
<hr><center>nginx/1.14.2</center>
</body>
</html>
)
```

Other commands (except sync, touch...) working fine. Mkdir for example:
```
rclone mkdir mailrue:backup/123 -vv
2020/09/20 00:59:43 DEBUG : rclone: Version "v1.53.1" starting with parameters ["rclone" "mkdir" "mailrue:backup/123" "-vv"]
2020/09/20 00:59:43 DEBUG : Creating backend with remote "mailrue:backup/123"
2020/09/20 00:59:43 DEBUG : Using config file from "/home/dabuka/.config/rclone/rclone.conf"
2020/09/20 00:59:43 DEBUG : Creating backend with remote "mailru:backup/hrvj2jq72i0s6s6o5am1khefn8/620k4b5vff2gg72cno2nfcb3tc"
2020/09/20 00:59:43 DEBUG : Encrypted drive 'mailrue:backup/123': Making directory
2020/09/20 00:59:43 DEBUG : [backup/hrvj2jq72i0s6s6o5am1khefn8/620k4b5vff2gg72cno2nfcb3tc]: new meta server: https://cld-extapi1.datacloudmail.ru/meta/
2020/09/20 00:59:43 DEBUG : 8 go routines active

rclone lsd mailrue:backup
           0 1970-01-01 03:00:00        -1 123
 39274358081 1970-01-01 03:00:00        -1 Documents
 28519623667 1970-01-01 03:00:00        -1 Music
 89973742307 1970-01-01 03:00:00        -1 Pictures
```

**Comment by darthShadow** [comment]
Can you please try again with the latest beta build? I believe this was fixed recently.

**Comment by Dabuka** [comment]
Thank you. This has been already fixed in v1.54.0-beta.4778.e2a57182b. Sorry for bothering.
